website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -176,4 +176,64 @@ Inline alerts can be added to a section or component or inline with content.
 
 Compact alerts can be added to a section or component or inline with content.
 
-![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
\ No newline at end of file
+![Placement of compact Alert](/assets/components/alert/alert-placement-compact.png =600x*)
+
+## Multiple alerts
+
+In general, displaying multiple alerts on the same page is discouraged as it can negatively impact the page hierarchy and unnecessarily shift emphasis away from the main content on the page.
+
+However, depending on the logic within the application and the type content being expressed to the user, it may be necessary to display more than one Alert on a page. Consider these scenarios and their accompanying guidelines if this is the case.
+
+### Alerts of the same color
+
+!!! Dont
+
+Don’t display more than one of the same `color` Alert on the page. This adds unnecessary visual weight to page and may result in the user being confused about which Alert is more important or which might require attention first.
+
+![Example of multiple Alerts of the same color](/assets/components/alert/multiple-color-alerts-dont.png)
+!!!
+
+!!! Do
+
+Instead, try to combine relevant and overlapping content into a single Alert.
+
+![Example of multiple Alerts combined into one](/assets/components/alert/multiple-color-alerts-do.png)
+
+This may result in the length of the content being longer, but consolidating messages into a single instance of the Alert is a more concise communication pattern.
+!!!
+
+### Ordering of Alerts
+
+Order multiple Alerts by their importance and impact on the user, with `critical` communicating Alerts that must be resolved immediately, and descending in perceived impact to `neutral`.
+
+![Ordering alerts](/assets/components/alert/ordering-alerts.png)
+
+### Alerts of the same type
+
+Try to avoid multiple Alerts of the same type (`page`, `inline`, etc) on the same page.
+
+!!! Do
+
+Instead, use Alerts hierarchically according to their intended prominence and importance to the user.
+
+![An example of multiple types of Alerts on the same page](/assets/components/alert/multiple-alert-types-do.png)
+!!!
+
+### Contextualizing Alerts
```

Comment:
This section does feel a little duplicative of the placement section, but the content is good. There may be value in merging the two into one section.

Reply:
I did a bit of reorganizing here to fit this content together, though there's not really a super clean way of doing this. The upside is now relevant content is combined, but the downside is now the info on placement is a bit more buried. This might not be an issue given the naming conventions we use, but just noting it here.

Reply:
Ah, I see. I had imagined that the content in "Contextualizing Alerts" would move into the Placement section (and the Placement section would remain where it was) as the placement content feels more important than multiple alerts. 

What if you dropped the heading "Contextualizing Alerts" and moved the content from that section directly into the "Placement" section? The two sections are trying to accomplish the same thing, but "placement" is a bit easier to understand as a concept. Let me know if you want to pair, happy to show you what I was thinking.